301-eventhub-namespace-ipfilter/azuredeploy.json

```diff
@@ -27,14 +28,22 @@
 			"metadata": {
 				"description": "IP Mask"
 			}
+		},
+		"Location": {
+			"type": "string",
+			"defaultValue": "East US2",
```

Comment:
use resourceGroup().location here and then override it in the parameters file...  Can also use "allowedValues" here if the list is small.

Reply:
resolved